commerce_option.install

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Drupal\views\Views;
+
+/**
+ * Implements hook_install().
+ */
+function commerce_option_install() {
+  $view = Views::getView('commerce_cart_form');
```

Comment:
Why are we changing views via hook_install? Couldn't we changing it via the UI and export out the view?

Reply:
The reason it that we don't provide the view in the commerce option module, It comes from commerce. If I just export the view with the new field (or even without changes) in our module it will error out ("already exists in active configuration") on commerce option module install.

We can export it in the config/sync folder but that is not part of the module. So the only way i know is to update the view programmatically on install. Let me know if I missed something

Reply:
Makes sense. Thanks.